app/src/main/java/com/duycuong/weather/utils/BindingUtils.java

```diff
@@ -4,20 +4,46 @@
 import android.support.design.widget.TabLayout;
 import android.support.v4.view.PagerAdapter;
 import android.support.v4.view.ViewPager;
+import android.support.v7.widget.RecyclerView;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import com.duycuong.weather.ui.screen.main.nextdays_weather.NextDaysAdapter;
+
+import java.math.RoundingMode;
+import java.text.DecimalFormat;
 
 /**
  * Created by DuyCương on 01/02/2018.
  */
 
 public class BindingUtils {
 
-    @BindingAdapter({ "bind:pagerAdapter" })
+    @BindingAdapter({"bind:pagerAdapter"})
     public static void setViewPagerAdapter(ViewPager viewPager, PagerAdapter adapter) {
         viewPager.setAdapter(adapter);
     }
 
-    @BindingAdapter({ "bind:pager" })
+    @BindingAdapter({"bind:pager"})
     public static void bindViewPagerTabs(final TabLayout view, final ViewPager pagerView) {
         view.setupWithViewPager(pagerView, true);
     }
+
+    @BindingAdapter({"bind:iconWeather"})
+    public static void loadImage(ImageView imageView, int idImage) {
+        imageView.setImageResource(idImage);
+    }
+
+    @BindingAdapter({"bind:roundingTemperature"})
+    public static void rounding(TextView textview, double temparature) {
+        DecimalFormat df = new DecimalFormat("#");
+        df.setRoundingMode(RoundingMode.CEILING);
+        textview.setText(df.format(temparature) + (char) 0x00B0);
```

Comment:
WARNING: [SetTextI18n] Do not concatenate text displayed with `setText`. Use resource string with placeholders.
 When calling `TextView#setText`
* Never call `Number#toString()` to format numbers; it will not handle fraction separators and locale-specific digits properly. Consider using `String#format` with proper format specifications (`%d` or `%f`) instead.
* Do not pass a string literal (e.g. "Hello") to display text. Hardcoded text can not be properly translated to other languages. Consider using Android resource strings instead.
* Do not build messages by concatenating text chunks. Such messages can not be properly translated.